README-1.8.md

```diff
@@ -0,0 +1,157 @@
+1. A sample script called `simulate-mfg.sh` to run the SDO manufacturing components (SCT - Supply Chain Tools) on a test VM device to initialize it with SDO, create the voucher, and extend it to the customer/owner. This script performs the same steps that a real SDO-enabled device manufacturer would.
+1. A script called `owner-boot-device` that initiates the same SDO booting process on a test VM device that runs on a physical SDO-enabled device when it boots.
+
+## Using the SDO Support
+
+Perform the following steps to try out the Horizon SDO support:
+
+- [Start the SDO Owner Services](#start-services) (only has to be done the first time)
+- [Initialize a Device with SDO](#init-device)
+- [Import the Ownership Voucher](#import-voucher)
+- [Boot the Device to Have it Configured](#boot-device)
+
+### <a name="start-services"></a>Start the SDO Owner Services
+
+**Todo: This section can be removed once the SDO owner services are being deployed in the mgmt hub.
+
+The SDO owner services respond to booting devices and enable administrators to import ownership vouchers. These 4 services are provided:
+
+- **RV**: A development version of the rendezvous server, the initial service that every SDO-enabled booting device contacts. The RV redirects the device to the OPS associated with the correct Horizon management hub.
+- **OPS**: The Owner Protocol Service communicates with the devices and securely downloads the device configuration scripts and files.
+- **OCS**: The Owner Companion Service manages the database files that contain the device configuration information.
+- **OCS-API**: A REST API that enables importing and querying ownership vouchers.
+
+The SDO owner services are packaged as a single docker container that can be run on any server that has network access to the Horizon management hub, and that the SDO devices can reach over the network.
+
+1. Get `run-sdo-owner-services.sh`, which is used to start the container:
+
+  ```bash
+  mkdir $HOME/sdo; cd $HOME/sdo
+  curl -sSLO https://raw.githubusercontent.com/open-horizon/SDO-support/stable/docker/run-sdo-owner-services.sh
+  chmod +x run-sdo-owner-services.sh
+  ```
+
+2. Run `./run-sdo-owner-services.sh -h` to see the usage, and set all of the necessary environment variables.
+
+3. Start the SDO owner services docker container and view the log:
+
+  ```bash
+  ./run-sdo-owner-services.sh
+  docker logs -f sdo-owner-services
+  ```
+
+#### Verify the SDO Owner Services API Endpoints
+
+**On a Horizon "admin" host** run these simple SDO APIs to verify that the services within the docker container are accessible and responding properly. (A Horizon admin host is one that has the `horizon-cli` package, which provides the `hzn` command, and has the environment variables `HZN_EXCHANGE_URL`, `HZN_SDO_SVC_URL`, `HZN_ORG_ID`, and `HZN_EXCHANGE_USER_AUTH` set correctly for your Horizon management hub.)
+
```

Comment:
1. Export your environment variables for the subsequent steps.

   ```bash
   export HZN_ORG_ID=mycluster
   export HZN_EXCHANGE_USER_AUTH=iamapikey:<api-key>
   export HZN_SDO_SVC_URL=http://<SDO_OWNER_SVC_HOST>:9008/api
   export SDO_RV_URL=http://<SDO_OWNER_SVC_HOST>:8040
```

Reply:
I agree that the above examples would be helpful ...and maybe if I would make one suggestion (because I recall you mentioning it to Ivan I believe), add the above but use lowercase `<sdo-owner-svcs-host>` to be consistent with the other documentation with how we've showed env vars vs something to substitute

Reply:
Both done.